Sources/BrowserServicesKit/Autofill/AutofillUserScript+SecureVault.swift

```diff
@@ -264,9 +267,10 @@ extension AutofillUserScript {
     }
     
     struct CredentialResponse: Codable {
-        let id: String
+        let id: String // <- TODO: when bitwarden is locked use id = "provider_locked"
```

Comment:
instead of "provider_locked", could you use a new property? For example providerStatus?